packages/account/tsconfig.json

```diff
@@ -3,19 +3,17 @@
     "compilerOptions": {
```

Comment:
what was the reason for changing these configs? it was just clean up or do we have any issue in our configs?
I'm asking this because we are fixing some TS configs and I want to make sure that everything is working properly.

Reply:
We can remove duplication of src by adding in baseUrl. So I made this change. the previous config didn't throw any error

Reply:
Will change this to use ur config ur new PR